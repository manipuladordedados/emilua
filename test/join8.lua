-- This test ensures join() detects EDEADLK

fib = spawn(function()
    coroutine.wrap(function()
        fib:join()
    end)()
end)
