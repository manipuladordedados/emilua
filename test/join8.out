Fiber 0x0 panicked: 'Resource deadlock avoided'
stack traceback:
	[string "?"]: in function <[string "?"]:0>
	input.lua:6: in function <input.lua:3>
